rust-mode.el

```diff
@@ -1528,6 +1606,7 @@ This is written mainly to be used as `end-of-defun-function' for Rust."
 (defvar rust-mode-map
   (let ((map (make-sparse-keymap)))
     (define-key map (kbd "C-c C-f") 'rust-format-buffer)
+    (define-key map (kbd "C-c C-n") 'rust-goto-format-problem)
```

Comment:
rather than adding a custom binding, is that possible to interface rust-goto-format-problem with next-error?